.travis.yml

```diff
@@ -5,6 +5,7 @@ node_js:
   - lts/*
   - '10'
 cache:
+  if: $(npm audit --audit-level=moderate) = 1
   npm: false
 before_install:
```

Comment:
[Q] Would we need to print something here that the cache has been cleared in case of error, for sanity in travis logs?

Reply:
Aren't travis logs verbose ? I guess it logs each command that's executed. I did have an echo statement for debugging purposes, I can add that back. Np

Reply:
Makes sense, shouldn't be required! :)